src/daemon/docker_exec.sh

```diff
@@ -24,41 +24,121 @@ function trap_error {
       echo "export $value" >> /root/.bashrc
     done
 
-    sleep 365d
+    sleep infinity
```

Comment:
I don't think this is helping to solve the purpose of that PR, or is it?
It feels like a nice addition but not mandatory to solve the root cause of the issue.

Reply:
True. I can split the commit to keep a trace of it.